Cargo.toml

```diff
@@ -134,6 +134,8 @@ serde_with = { version = "3.3.0", default-features = false }
 parity-scale-codec = { version = "3.6.5", default-features = false }
 json5 = "0.4.1"
 
+storage = { git = "https://github.com/Erigara/storage.git", branch="main" }
```

Comment:
Not sure whatever this crate should be part of iroha, used as building block for new-wsv.

Reply:
I'd suggest committing it to crates.io, and tidying it up a bit. As it stands it's not something that most people would be comfortable with.